doc/sphinx-apidoc/conf.py

```diff
@@ -235,9 +235,9 @@
 
 
 def setup(app):
```

Comment:
``custom.js`` does not exist and can be removed from here. Should we use the approach described in https://docs.readthedocs.io/en/stable/guides/adding-custom-css.html and remove the setup() function entirely?

Reply:
Thanks, done.